src/main/java/teammates/ui/output/FeedbackSessionData.java

```diff
@@ -241,6 +271,36 @@ public void setPrivileges(InstructorPermissionSet privileges) {
         this.privileges = privileges;
     }
 
+    public void setStudentDeadlines(Map<String, Long> studentDeadlines) {
+        this.studentDeadlines = studentDeadlines;
+    }
+
+    public void setInstructorDeadlines(Map<String, Long> instructorDeadlines) {
+        this.instructorDeadlines = instructorDeadlines;
+    }
+
+    /**
+     * Filter out all the deadlines that are not for the given student.
+     */
+    public void filterDeadlinesForStudent(String studentEmailAddress) {
+        setStudentDeadlines(studentDeadlines.entrySet()
+                .stream()
+                .filter(entry -> entry.getKey().equals(studentEmailAddress))
+                .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue)));
+        setInstructorDeadlines(new HashMap<>());
+    }
+
+    /**
+     * Filter out all the deadlines that are not for the given instructor.
+     */
+    public void filterDeadlinesForInstructor(String instructorEmailAddress) {
+        setStudentDeadlines(new HashMap<>());
+        setInstructorDeadlines(instructorDeadlines.entrySet()
+                .stream()
+                .filter(entry -> entry.getKey().equals(instructorEmailAddress))
+                .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue)));
+    }
```

Comment:
Any tests for these two methods?

Reply:
The maps are tested in the outputs in `GetFeedbackSessionActionTest`. One is checked for emptiness; the other is checked for if it contains that key, and also if it contains only one key.